packages/aila/src/helpers/chat/generateMessageId.ts

```diff
@@ -0,0 +1,9 @@
+import { nanoid } from "nanoid";
+
+export function generateMessageId({
+  role,
+}: {
+  role: "user" | "assistant" | "system";
+}) {
+  return `${role[0]}-${nanoid(16)}`;
```

Comment:
I distinguished between roles in development ... do we want to keep?

Reply:
nice